bin/deploy-vhost

```diff
@@ -246,7 +247,12 @@ sub apache_graceful {
 }
 
 sub nginx_graceful {
-    shell("/etc/init.d/nginx", "reload") if -e "/etc/init.d/nginx";
+    if ($init_system eq "systemd") {
+        shell("/bin/systemctl", "reload", "nginx");
```

Comment:
Does this need to check if the `nginx` service exists, similar to the `if -e "/etc/init.d/nginx"` test?

Reply:
It should really only get called where nginx is present. To check we'd probably need to call `systemctl status nginx` and look at the return code as the service unit file could potentially exist in a number of places, so it's not quite as easy to test for anyway.

I may make a subsequent commit to adjust `remove_site` to only call this when `"https": "nginx"` as well.